packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -150,30 +172,35 @@ export const DatepickerCalendar: ComponentWithAs<'div', DatepickerCalendarProps>
     design,
     styles,
     variables,
+    calendarHeaderCell,
+    calendarCell,
+    header,
     selectedDate,
     navigatedDate,
     firstDayOfWeek,
     firstWeekOfYear,
     dateRangeType,
+    showWeekNumbers,
   } = props;
 
-  const localizedStrings = props.localizedStrings ?? DEFAULT_CALENDAR_LOCALIZED_STRINGS;
+  const localizedStrings = props.localizedStrings || DEFAULT_CALENDAR_LOCALIZED_STRINGS;
   const ElementType = getElementType(props);
   const unhandledProps = useUnhandledProps(DatepickerCalendar.handledProps, props);
   const getA11yProps = useAccessibility(props.accessibility, {
     debugName: DatepickerCalendar.displayName,
     actionHandlers: {},
     rtl: context.rtl,
   });
-  const gridOptions: IDayGridOptions = {
-    selectedDate: selectedDate ?? props.today ?? new Date(),
-    navigatedDate: navigatedDate ?? props.today ?? new Date(),
+
+  const [gridOptions, setGridOptions] = React.useState<IDayGridOptions>({
+    selectedDate: selectedDate || props.today || new Date(),
+    navigatedDate: navigatedDate || props.today || new Date(),
     firstDayOfWeek,
     firstWeekOfYear,
     dateRangeType,
-  };
+  });
```

Comment:
Are there any reasons to store the whole object under `useState()`? As I see only `navigatedDate` can be modified